src/all.ts

```diff
@@ -173,8 +173,11 @@ import './icons/CheckCircleWhiteIcon';
 import './icons/CheckmarkDuotoneIcon';
 import './icons/CheckmarkIcon';
 import './icons/CheckmarkWhiteIcon';
+import './icons/CheckoutBlack1Icon';
+import './icons/CheckoutDuotone1Icon';
```

Comment:
What is going on here? Is the `1` intentional?

Reply:
Oh I need to rerun the command. For some reason, Amino was exported duplicates of the product icons so I had to remove 9 icons, but I did that afterwards. I'll fix it.